refinery/ui/.eslintignore

```diff
@@ -4,5 +4,3 @@ development/**
 docs/**
 node_modules/**
 production/**
-
-source/js/provvis/**
```

Comment:
I'll revert this, but...
- If we can get down to zero warnings, it makes it more likely that we'll keep it clean going forward
- it's probably more obvious to have a TODO at the top of the file that needs fixing, than to ignore a whole directory